src/Umbraco.Web/Unversion/UnversionService.cs

```diff
@@ -25,27 +25,31 @@ public UnversionService(IUnversionConfig config, ILogger logger, IUmbracoContext
 
         public void Unversion(IContent content)
         {
+            // Get configuration entries for this piece of content
             var configEntries = new List<UnversionConfigEntry>();
 
+            // Add all config entries if it matches our ContentType Alias
             if (_config.ConfigEntries.ContainsKey(content.ContentType.Alias))
                 configEntries.AddRange(_config.ConfigEntries[content.ContentType.Alias]);
 
+            // Add all config entries (would it be more than one though?)
             if (_config.ConfigEntries.ContainsKey(UnversionConfig.AllDocumentTypesKey))
                 configEntries.AddRange(_config.ConfigEntries[UnversionConfig.AllDocumentTypesKey]);
 
             if (configEntries.Count <= 0)
             {
-                _logger.Debug<UnversionService>("No unversion configuration found for type {alias}", content.ContentType.Alias);
+                // Would this ever get hit ?
+                _logger.Debug<UnversionService>("No unversion configuration found for type {ContentTypeAlias}", content.ContentType.Alias);
                 return;
             }
 
             foreach (var configEntry in configEntries)
             {
                 var isValid = true;
 
-                // Check the RootXPath if configured
+                // Check if RootXPath is configured
                 if (!string.IsNullOrEmpty(configEntry.RootXPath))
-                    // TODO: Fix in some otherway
+                    // TODO: Fix in some otherway (Check with Soren on what he meant by this)
```

Comment:
I don't really recall, but I could imagine that GetNodeIdsFromXpath was broken on the first try to port, and I probably just forgot to remove the comment :)

Reply:
OK I will remove the comment & assume this is all good then :)